flybywiresim/a32nx
# fix(bleed): fix erroneous fault light indication

Fixes #[issue_no]

## Summary of Changes

There was an issue introduced in #5384 that caused the engine bleed pushbutton fault light to come on when it should not.

This PR removes the fault light indication logic until the regulation is fixed such that the fault light will not come on in normal operations.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): BlueberryKing#6641

## Testing instructions

- Turn off the packs and run up the engines to TOGA and keep them there for at least a minute. Make sure the engine bleed fault light does not light up on the overhead panel.
- Restart the flight from the runway.
- Perform a take off with packs off.
- Make sure the fault light on the engine bleed pushbuttons stays off until you reach acceleration altitude.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
